go.mod

```diff
@@ -1,5 +1,10 @@
 module github.com/bsm/redislock
 
-go 1.13
+go 1.18
```

Comment:
Many thanks for this! But, could you please: 

1. `go mod tidy -go=1.17`, we would like to support the latest 2 go versions
2. rebase against main to incorporate https://github.com/bsm/redislock/pull/45 - this should fix CI

Will then merge ASAP. Thanks again!